### PR TITLE
Add gitpod config

### PR DESCRIPTION
**Pull request by antoine2711**
this commit adds support for Gitpod.io, a free automated
dev environment that makes contributing and generally working on GitHub
projects much easier. It allows anyone to start a ready-to-code dev
environment for any branch, issue and pull request with a single click.

### CODE REVIEW

**Comment by wetneb** [comment]
I suspect the Dockerfile you added will not be enough: don't you need to make a specific one, adapted to OpenRefine's stack and environment? Or can we try this gitpod set up from your fork?

File: .gitpod.Dockerfile [hunk 1]
```diff
@@ -0,0 +1,10 @@
+FROM gitpod/workspace-mysql
```
Comment (thadguidry) [inline .gitpod.Dockerfile]:
```suggestion
FROM maven:3.6.3-jdk-8-openj9
```

Comment (thadguidry) [reply .gitpod.Dockerfile]: Just Maven with the OpenJDK should be enough. https://www.gitpod.io/docs/config-docker/

**Comment by antoine2711** [comment]
Sorry for this. This is me trying the GitPod environment. Thanks or your input.

**Pull request closed without merging**